practicedsa.html

```diff
@@ -46,47 +46,48 @@
 <body>
 
     <section>
-        <nav class="navbar navbar-expand-lg navbar-dark fixed-top nav-col top">
-            <a class="navbar-brand" href="index.html"><img src="images/icon.png" alt="Algo Phantoms" width="50"
-                    height="50"><span class="brand1">Algo</span>Phantoms
-            </a>
+        <nav class="navbar navbar-expand-lg navbar-dark fixed-top nav-col top" id='nav'>
+            <a class="navbar-brand" href="index.html"><img src="images/icon.png" alt="Algo Phantoms" width="50" height="50"><strong><span class="brand1">Algo</span>Phantoms</strong></a>
             <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent"
                 aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
                 <span class="navbar-toggler-icon"></span>
             </button>
 
             <div class="collapse navbar-collapse" id="navbarSupportedContent">
                 <ul class="navbar-nav ml-auto">
-                    <li class="nav-item">
+                    <li class="nav-item active">
                         <a class="nav-link font-weight-bold nav-items" href="index.html">Home </a>
+                                <hr class="hr">
                     </li>
                     <li class="nav-item">
                         <a class="nav-link font-weight-bold nav-items" href="index.html#about">About</a>
+                        <hr class="hr">
                     </li>
 
                     <li class="nav-item">
-                        <a class="nav-link  font-weight-bold nav-items" href="team.html">Team<span
-                                class="sr-only">(current)</span></a>
+                        <a class="nav-link  font-weight-bold nav-items" href="team.html">Team</a>
+                        <hr class="hr">
                     </li>
                     
                     <li class="nav-item dropdown">
                         <a class="nav-link font-weight-bold nav-items dropdown-toggle" href="#" id="navbarDropdown"
                             role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-                            More
-                        </a>
+                            More</a>
+                            <hr class="hr">
                         <div class="dropdown-menu dropdown-menu-center" aria-labelledby="navbarDropdown">
                             <a class="dropdown-item" href="DSA.html">RoadMap to your DSA</a>
 
                             <a class="dropdown-item" href="https://github.com/Algo-Phantoms" target="_blank">Contribute
                                 to Project</a>
 
-                            <a class="dropdown-item" href="practicedsa.html">DSA Problems</a>
-
+                            <a class="dropdown-item" href="#">DSA Problems</a>
```

Comment:
Add `active` class and `<span  class="sr-only">(current)</span></a>` to this as it is the active page.